### PR TITLE
SOV-2478: Documentation for Atoms in Gitbook (part 2)

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-2478
fixes in this PR:
 - move Toggle component to Atoms section
 - fix broken "All Variations" section in the Input component

### CODE REVIEW

**Review by creed-victor — APPROVED** [review]

File: packages/ui/src/1_atoms/HealthBar/HealthBar.stories.tsx [hunk 1]
```diff
@@ -25,6 +25,37 @@ Default.args = {
...
+    control: 'number',
+    description: 'The value of the bar',
+  },
+  className: {
+    control: 'text',
+    description: 'The className to apply to the bar',
+  },
+  dataAttribute: {
+    control: 'text',
+    description: 'The data attribute to apply to the bar',
```
Comment (soulBit) [inline packages/ui/src/1_atoms/HealthBar/HealthBar.stories.tsx]:
```suggestion
    description: 'The data id to apply as HTML attribute to this component instance. This should be unique per component instance on the page',
```

Please update to the suggested, and you can also reuse this for any other dataAttribute props we have in other components (saves having to customise it each time)

File: packages/ui/src/1_atoms/Input/Input.stories.tsx [hunk 2]
```diff
@@ -50,17 +50,71 @@ TextInput.args = {
...
+    control: 'function',
+    description: 'The onChange handler for the input',
+  },
+  classNameInput: {
+    control: 'text',
+    description: 'The className to apply to the input',
+  },
+  invalid: {
+    control: 'boolean',
+    description: 'Whether the input is invalid',
```
Comment (soulBit) [inline packages/ui/src/1_atoms/Input/Input.stories.tsx]:
```suggestion
    description: 'Whether the input is invalid. When set to true the error state styling and behaviour will be triggered on the input',
```

File: packages/ui/src/1_atoms/Input/Input.stories.tsx [hunk 3]
```diff
@@ -50,17 +50,71 @@ TextInput.args = {
...
+    control: 'text',
+    description: 'The data attributes to apply to the input',
+  },
+  debounce: {
+    control: 'number',
+    description: 'The debounce time in ms',
+  },
+  onChangeText: {
+    control: 'function',
+    description: 'The onChange handler for the input',
```
Comment (soulBit) [inline packages/ui/src/1_atoms/Input/Input.stories.tsx]:
```suggestion
    description: 'The onChange handler for the input, triggered whenever the input's value changes',
```

File: packages/ui/src/1_atoms/Icon/Icon.stories.tsx [hunk 4]
```diff
@@ -79,16 +79,88 @@ Basic.args = {
...
+  ...Basic.argTypes,
+  icon: {
+    control: 'string',
+    defaultValue: faBacterium,
+    description:
+      'The name of the icon to display. It can be the name of a Sovryn UI icon, a custom SVG element, or an imported Fontawesome icon.',
+  },
+  size: {
+    control: 'select',
+    options: [
```
Comment (soulBit) [inline packages/ui/src/1_atoms/Icon/Icon.stories.tsx]: Can we pull these dynamically from `SizeProp`?

File: packages/ui/src/1_atoms/Input/Input.stories.tsx [hunk 5]
```diff
@@ -50,17 +50,71 @@ TextInput.args = {
...
 
 export const DebouncedInput = AdvancedTemplate.bind({});
 DebouncedInput.args = {
   debounce: 300,
 };
+DebouncedInput.argTypes = {
+  ...TextInput.argTypes,
+  debounce: {
+    control: 'number',
+    description: 'The debounce time in ms',
```
Comment (soulBit) [inline packages/ui/src/1_atoms/Input/Input.stories.tsx]:
```suggestion
    description: 'The debounce time in milliseconds',
```

File: packages/ui/src/1_atoms/Lottie/Lottie.stories.tsx [hunk 6]
```diff
@@ -54,6 +54,80 @@ Single.args = {
...
+    control: 'select',
+    options: Object.keys(animationMap),
+    defaultValue: 'spaceScene',
+    description: 'Choose the animation to display',
+  },
+  loop: {
+    control: 'select',
+    options: [true, 1, 2, 3, 4, 5],
+    defaultValue: true,
+    description: 'Set the number of times to loop the animation',
```
Comment (soulBit) [inline packages/ui/src/1_atoms/Lottie/Lottie.stories.tsx]:
```suggestion
    description: 'Set the number of times to loop the animation. Loops indefinitely when set to "true"',
```

File: packages/ui/src/1_atoms/Paragraph/Paragraph.stories.tsx [hunk 7]
```diff
@@ -81,6 +81,34 @@ const Template: Story<ComponentProps<typeof Paragraph>> = () => (
 
 export const Default = Template.bind({});
 
+Default.argTypes = {
+  children: {
+    control: 'text',
+    description:
+      'The content of the paragraph. Can be text, other components, or HTML elements',
```
Comment (soulBit) [inline packages/ui/src/1_atoms/Paragraph/Paragraph.stories.tsx]:
```suggestion
      'The content of the paragraph. Can be text, other components, or HTML elements. Underlying DOM element is a Paragraph tag so children should be compliant with this',
```

**Review by soulBit — CHANGES_REQUESTED** [review]

File: packages/ui/src/1_atoms/Icon/Icon.stories.tsx (hunk 4 above)
Comment (pietro-maximoff) [reply packages/ui/src/1_atoms/Icon/Icon.stories.tsx]: The SizeProp type itself - its a fixed set of string values and I created a separate array to hold those values and refer to them.

Comment (soulBit) [reply packages/ui/src/1_atoms/Icon/Icon.stories.tsx]: ah I see, we don't actually support all sizes then? from the Icon component code it appears like we do

**Review by soulBit — APPROVED** [review]

**Review by soulBit — APPROVED** [review]

**Pull request merged**